cluster-autoscaler/utils/daemonset/daemonset.go

```diff
@@ -33,14 +33,14 @@ const (
 )
 
 // GetDaemonSetPodsForNode returns daemonset nodes for the given pod.
-func GetDaemonSetPodsForNode(nodeInfo *schedulerframework.NodeInfo, daemonsets []*appsv1.DaemonSet) ([]*apiv1.Pod, error) {
-	result := make([]*apiv1.Pod, 0)
+func GetDaemonSetPodsForNode(nodeInfo *framework.NodeInfo, daemonsets []*appsv1.DaemonSet) ([]*framework.PodInfo, error) {
+	result := make([]*framework.PodInfo, 0)
 	for _, ds := range daemonsets {
 		shouldRun, _ := daemon.NodeShouldRunDaemonPod(nodeInfo.Node(), ds)
 		if shouldRun {
 			pod := daemon.NewPod(ds, nodeInfo.Node().Name)
 			pod.Name = fmt.Sprintf("%s-pod-%d", ds.Name, rand.Int63())
-			result = append(result, pod)
+			result = append(result, &framework.PodInfo{Pod: pod})
```

Comment:
Similar to above and probably plenty of other places -> should we change this to somehow include DRA claims (where would we get them from though)?

Reply:
Yes, all this logic will be added in DRA-specific PRs. Though this particular place will probably be a TODO in the initial implementation (meaning that forcing missing DS pods that use DRA won't work initially, but that's probably not a popular use-case).